### PR TITLE
Update sample model submission for new submission processing

### PR DESCRIPTION
**Pull request by franzigeiger**
- Add bibtex method
- Move model test to model-tools
- Add tensorflow example

### CODE REVIEW

File: brainscore_model_template.iml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="PYTHON_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
```
Comment (mschrimpf) [inline brainscore_model_template.iml:9]: is this a necessary file? can probably be deleted from git

File: examples/keras.py [hunk 2]
```diff
@@ -25,6 +25,14 @@ def get_layers(name):
     return [f'block{i + 1}_pool' for i in range(5)] + ['fc1', 'fc2']
 
 
+def get_bibtex(model_identifier):
+    return """@InProceedings{Simonyan15,
+              author       = "Karen Simonyan and Andrew Zisserman",
+              title        = "Very Deep Convolutional Networks for Large-Scale Image Recognition",
+              booktitle    = "International Conference on Learning Representations",
+              year         = "2015",
```
Comment (mschrimpf) [inline examples/keras.py:33]: missing `url` parameter

File: examples/pytorch.py [hunk 3]
```diff
@@ -30,5 +33,17 @@ def get_layers(name):
             'classifier.2', 'classifier.5']
 
 
+def get_bibtex(model_identifier):
+    return """@incollection{NIPS2012_4824,
```
Comment (mschrimpf) [inline examples/pytorch.py:37]: maybe add an `assert model_identifier == ...` to make it clear that the bibtex can depend on the `model_identifier`

File: examples/tensorflow.py [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+# If you use tensorflow, don't forget to add it and its dependencies to the setup.py
+def get_model_list():
+    return ['resnet50']
+
+
+def get_model(name):
+    assert name == 'resnet50'
+    TFSlimModel.init('resnet-50_v1', net_name='resnet_v1_50', preprocessing_type='vgg',
+                     image_size=224, labels_offset=0)
+    model = keras.applications.vgg16.VGG16()
```
Comment (mschrimpf) [inline examples/tensorflow.py:15]: there's code for both resnet-50 and vgg16 here

File: models/base_models.py [hunk 5]
```diff
@@ -38,5 +39,15 @@ def get_layers(name):
     """
     return []
 
+
+def get_bibtex(model_identifier):
+    """
+    A method returning the bibtex reference of the requested model as a string.
+    """
+    return ''
```
Comment (mschrimpf) [inline models/base_models.py:47]: could also make this into `raise NotImplementedError()` and catch in the submission system to make it more clear

File: models/brain_models.py [hunk 6]
```diff
@@ -27,5 +26,14 @@ def get_model(name):
     return
 
 
+def get_bibtex(model_identifier):
+    """
+    A method returning the bibtex reference of the requested model as a string.
+    """
+    return ''
```
Comment (mschrimpf) [inline models/brain_models.py:33]: as above with the `raise`

File: models/base_models.py (hunk 5 above)
Comment (franzigeiger) [reply models/base_models.py:47]: hm I handled the empty string in brain-score so I would leave that, same below

**Review by mschrimpf — APPROVED** [review]

**Pull request merged**